eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -864,10 +865,10 @@ class BabbageEraPParams era => ConwayEraPParams era where
   hkdDRepVotingThresholdsL :: HKDFunctor f => Lens' (PParamsHKD f era) (HKD f DRepVotingThresholds)
   hkdCommitteeMinSizeL :: HKDFunctor f => Lens' (PParamsHKD f era) (HKD f Natural)
   hkdCommitteeMaxTermLengthL :: HKDFunctor f => Lens' (PParamsHKD f era) (HKD f EpochNo)
```

Comment:
I also just realized that I forgot to list `hkdCommitteeMaxTermLengthL` in the ticket, but it also needs to change, since it is not an absolute epoch number
```suggestion
  hkdCommitteeMaxTermLengthL :: HKDFunctor f => Lens' (PParamsHKD f era) (HKD f EpochInterval)
```